wfc3tools/wf3cte.py

```diff
@@ -105,12 +36,8 @@ def wf3cte(input, output=None, parallel=True, verbose=False, log_func=print):
         - a single filename (``iaa012wdq_raw.fits``)
         - a Python list of filenames
         - a partial filename with wildcards (``*raw.fits``)
-        - filename of an ASN table (``*asn.fits``)
         - an at-file (``@input``)
 
-    output : str, default=None
```

Comment:
also curious why this option was removed?

Reply:
This is the same answer as above.  The "output" parameter was improperly added as a user option, but the underlying code actually constructs the output filenames.

Reply:
Addressed in 6101480.